.github/workflows/unittests_ci.yaml

```diff
@@ -0,0 +1,23 @@
+name: Unit testing scanpy_scripts
+
+on: [push, pull_request]
+
```

Comment:
Actually, could you add the testing to the existing suite? https://github.com/ebi-gene-expression-group/scanpy-scripts/blob/develop/.github/workflows/python-package.yml

You'll see that the dependencies etc are already in place there.

Reply:
Done in 0d4ad70cb2147286042854714504580eec446f7b.